Automattic/newspack-theme
# Add telephone icon to social menu

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a smartphone icon to the social links menu, so when you add a link to a phone number (prefixing it with `tel:`), it will display as a smartphone.

Closes #843

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Add a telephone number to your social links menu, prefixing with `tel:`
3. Confirm that it displays as a smartphone:

![image](https://user-images.githubusercontent.com/177561/77473748-22f17d00-6dd3-11ea-8af6-378d05893800.png)

### Other information:
